main.py

```diff
@@ -1,10 +1,40 @@
+import json
+
 import init_django_orm  # noqa: F401
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+    with open("players.json", "r") as file:
+        players = json.load(file)
+
+        for player, data in players.items():
```

Comment:
![image](https://github.com/mate-academy/py-game-models/assets/108459229/c23e8a97-5e5b-4dd9-a78c-e699bf2b6a4e)

Reply:
Thanks. Fixed that